frontend/src/app/modules/scripts/script-result/script-result.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="script-result">
+    <p class="result-message">
+        Last result from query "{{ scriptTitle }}"
+        at {{ scriptResult.date | date : 'medium' }}:
```

Comment:
it shouldn't be bold if any (any redundant `bold` there?

Reply:
made regular